test/bin/scenario.sh

```diff
@@ -38,6 +38,48 @@ full_vm_name() {
     echo "${SCENARIO//@/-}-${base}"
 }
 
+# hostname validation
+# based on https://github.com/rhinstaller/anaconda/blob/c95142f76735a2e9ae6d845f8569d46632ddd619/pyanaconda/network.py#L96-L120
+validate_full_vm_name() {
+    local vm_name="$1"
+
+    if [ -z "${vm_name}" ]; then
+        error "VM hostname cannot be empty string" 
+        record_junit "${vmname}" "vm_name_validation" "FAILED"
```

Comment:
Similar thing in other places
```suggestion
        record_junit "${vm_name}" "vm_name_validation" "FAILED"
```